README.md

```diff
@@ -1 +1,19 @@
-# svccat
\ No newline at end of file
+# svccat
+
+This is a collection of materials intended to educate about the use of [Service Catalog](https://github.com/kubernetes-incubator/service-catalog), a
+Kubernetes extension project that implements the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker). This allows
```

Comment:
Link specifically to the README.md https://github.com/openservicebrokerapi/servicebroker/blob/master/README.md